### ISSUE TITLE
I quit a program running a background timer task.

### ISSUE THREAD

**Issue by pnachtwey** [spyder-ide/spyder#12170]
### What steps will reproduce the problem?

I used the quit open when I right click on the console part of the screen

### Traceback
```python-traceback
  File "C:\Anaconda3\lib\site-packages\psutil\_pswindows.py", line 636, in wrapper
    return fun(self, *args, **kwargs)
  File "C:\Anaconda3\lib\site-packages\psutil\_pswindows.py", line 847, in create_time
    return cext.proc_create_time(self.pid)
ProcessLookupError: [Errno 3] No such process

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Anaconda3\lib\site-packages\psutil\__init__.py", line 365, in _init
    self.create_time()
  File "C:\Anaconda3\lib\site-packages\psutil\__init__.py", line 696, in create_time
    self._create_time = self._proc.create_time()
  File "C:\Anaconda3\lib\site-packages\psutil\_pswindows.py", line 641, in wrapper
    raise NoSuchProcess(self.pid, self._name)
psutil._exceptions.NoSuchProcess: psutil.NoSuchProcess process no longer exists (pid=21644)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Anaconda3\lib\site-packages\qtconsole\manager.py", line 27, in poll
    super(QtKernelRestarter, self).poll()
  File "C:\Anaconda3\lib\site-packages\jupyter_client\restarter.py", line 113, in poll
    self.kernel_manager.restart_kernel(now=True, newports=newports)
  File "C:\Anaconda3\lib\site-packages\jupyter_client\manager.py", line 361, in restart_kernel
    self.shutdown_kernel(now=now, restart=True)
  File "C:\Anaconda3\lib\site-packages\jupyter_client\manager.py", line 321, in shutdown_kernel
    self._kill_kernel()
  File "C:\Anaconda3\lib\site-packages\spyder\plugins\ipythonconsole\utils\manager.py", line 78, in _kill_kernel
    self.kill_proc_tree(self.kernel.pid)
  File "C:\Anaconda3\lib\site-packages\spyder\plugins\ipythonconsole\utils\manager.py", line 44, in kill_proc_tree
    parent = psutil.Process(pid)
  File "C:\Anaconda3\lib\site-packages\psutil\__init__.py", line 338, in __init__
    self._init(pid)
  File "C:\Anaconda3\lib\site-packages\psutil\__init__.py", line 378, in _init
    raise NoSuchProcess(pid, None, msg)
psutil._exceptions.NoSuchProcess: psutil.NoSuchProcess no process found with pid 21644
WARNING:traitlets:kernel died: 45.00063943862915
```

## Versions

* Spyder version: 4.1.1 
* Python version: 3.7.1
* Qt version: 5.9.6
* PyQt5 version: 5.9.2
* Operating System: Windows 10

### Dependencies

```

# Mandatory:
atomicwrites >=1.2.0           :  1.2.1 (OK)
chardet >=2.0.0                :  3.0.4 (OK)
cloudpickle >=0.5.0            :  0.6.1 (OK)
diff_match_patch >=20181111    :  20181111 (OK)
intervaltree                   :  None (OK)
IPython >=4.0                  :  7.2.0 (OK)
jedi =0.15.2                   :  0.15.2 (OK)
nbconvert >=4.0                :  5.4.0 (OK)
numpydoc >=0.6.0               :  0.8.0 (OK)
paramiko >=2.4.0               :  2.4.2 (OK)
parso =0.5.2                   :  0.5.2 (OK)
pexpect >=4.4.0                :  4.7.0 (OK)
pickleshare >=0.4              :  0.7.5 (OK)
psutil >=5.3                   :  5.4.8 (OK)
pygments >=2.0                 :  2.3.1 (OK)
pylint >=0.25                  :  2.2.2 (OK)
pyls >=0.31.9;<0.32.0          :  0.31.9 (OK)
qdarkstyle >=2.8               :  2.8 (OK)
qtawesome >=0.5.7              :  0.6.0 (OK)
qtconsole >=4.6.0              :  4.6.0 (OK)
qtpy >=1.5.0                   :  1.5.2 (OK)
rtree >=0.8.3                  :  0.8.3 (OK)
sphinx >=0.6.6                 :  1.8.2 (OK)
spyder_kernels >=1.9.0;<1.10.0 :  1.9.0 (OK)
watchdog                       :  None (OK)
zmq >=17                       :  17.1.2 (OK)

# Optional:
cython >=0.21                  :  0.29.2 (OK)
matplotlib >=2.0.0             :  3.0.2 (OK)
numpy >=1.7                    :  1.15.4 (OK)
pandas >=0.13.1                :  0.23.4 (OK)
scipy >=0.17.0                 :  1.1.0 (OK)
sympy >=0.7.3                  :  1.3 (OK)
```

**Comment by ccordoba12** [comment]
Duplicate of #11872

**Comment by ccordoba12** [comment]
It'll be fixed in our next version (4.1.2), to be released shortly.
